sdk/src/main/java/com/mapbox/maps/MapSurface.kt

```diff
@@ -49,6 +50,7 @@ class MapSurface @JvmOverloads constructor(
    */
   fun surfaceCreated() {
     renderer.surfaceCreated()
+    mapController.setScreenRefreshRate((context.getSystemService(Context.WINDOW_SERVICE) as WindowManager).defaultDisplay.refreshRate)
```

Comment:
How about multi-display use cases?

Reply:
In a typical Android Auto use case, the MapSurface is not the default display, and does not necessarily have the same refresh rate as the primary display.

Reply:
@pengdev I did not manage to actually simulate primary display with one refresh rate and secondary with another but I did make sure that applying different user refresh rates (via `setMaximumFps`) to map on primary and secondary displays works fine. Guess that's all we could do now, if some extraordinary case will pop-up (although it's indeed very specific use-case but it should be handled).

Also I did check via our `SurfaceActivity` that methods `mapSurface.setMaximumFps` and `mapSurface.setOnFpsChangedListener` do work as expected.